content/_index.md

```diff
@@ -12,7 +12,7 @@ It is intended to be the hub for all things related to the Kubernetes Contributo
 
 The first step is to check out the [contributor guide](/guide/). All contributors must abide by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) and [sign the CLA](https://github.com/kubernetes/community/blob/master/CLA.md).
 
-If you need help with submitting a pull request, found a bug, or have a suggestion, please join us in [Slack] and pop over to the [#kubernetes-dev channel][kdev].
+If you need help with submitting a pull request, found a bug, or have a suggestion, please join us in [Slack]((https://slack.k8s.io/)) and pop over to the [#kubernetes-dev channel](https://app.slack.com/client/T09NY5SBT/C09R23FHP).
```

Comment:
Is there a reason you switched from the reference style links? In most cases they're preferred over inline.